pkg/applyconfigurations/testdata/ac/zz_generated.applyconfigurations.go

```diff
@@ -0,0 +1,441 @@
+	b.BinaryName = &value
+	return b
+}
+
+// WithCanBeNull sets the CanBeNull field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithCanBeNull(value string) *CronJobSpecApplyConfiguration {
+	b.CanBeNull = &value
+	return b
+}
+
+// WithJobTemplate sets the JobTemplate field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithJobTemplate(value *v1beta1.JobTemplateSpecApplyConfiguration) *CronJobSpecApplyConfiguration {
+	b.JobTemplate = value
+	return b
+}
+
+// WithSuccessfulJobsHistoryLimit sets the SuccessfulJobsHistoryLimit field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithSuccessfulJobsHistoryLimit(value *int32) *CronJobSpecApplyConfiguration {
+	b.SuccessfulJobsHistoryLimit = value
+	return b
+}
+
+// WithFailedJobsHistoryLimit sets the FailedJobsHistoryLimit field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithFailedJobsHistoryLimit(value *int32) *CronJobSpecApplyConfiguration {
+	b.FailedJobsHistoryLimit = value
+	return b
+}
+
+// WithByteSliceData sets the ByteSliceData field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithByteSliceData(value map[string][]byte) *CronJobSpecApplyConfiguration {
+	b.ByteSliceData = &value
+	return b
+}
+
+// WithStringSliceData sets the StringSliceData field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithStringSliceData(value map[string][]string) *CronJobSpecApplyConfiguration {
+	b.StringSliceData = &value
+	return b
+}
+
+// WithPtrData sets the PtrData field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithPtrData(value map[string]*string) *CronJobSpecApplyConfiguration {
+	b.PtrData = &value
+	return b
+}
+
+// WithTwoOfAKindPart0 sets the TwoOfAKindPart0 field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithTwoOfAKindPart0(value string) *CronJobSpecApplyConfiguration {
+	b.TwoOfAKindPart0 = &value
+	return b
+}
+
+// WithTwoOfAKindPart1 sets the TwoOfAKindPart1 field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithTwoOfAKindPart1(value testdata.LongerString) *CronJobSpecApplyConfiguration {
+	b.TwoOfAKindPart1 = &value
+	return b
+}
+
+// WithDefaultedString sets the DefaultedString field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithDefaultedString(value string) *CronJobSpecApplyConfiguration {
+	b.DefaultedString = &value
+	return b
+}
+
+// WithDefaultedSlice sets the DefaultedSlice field in the declarative configuration to the given value
+func (b *CronJobSpecApplyConfiguration) WithDefaultedSlice(value []string) *CronJobSpecApplyConfiguration {
```

Comment:
Vararg slices? E.g. https://github.com/kubernetes/kubernetes/blob/7e2b5f3325a2f7808acd6740e9f0d907976e4704/staging/src/k8s.io/client-go/applyconfigurations/apps/v1/replicasetstatus.go#L81

Reply:
Note that for slices we append and for maps we put in "With" calls.